### PR TITLE
Add a text that explains why the team contest option is disabled

### PR DESCRIPTION
**Pull request by iqbalcodes6602**
# Descripción

I have added the text 'This option is only available for new competitions.' to already exiting description.

Si tu cambio toca la interfaz, hay que incluir una captura de pantalla.

Fixes: #6961 

# Comentarios

Si hay comentarios para los reviewers (por ejemplo, dónde hay que empezar a
revisar, algo a lo que hay que poner atención adicional, etc.), todo eso se
puede poner en esta sección.

También si faltan cosas por hacer, se puede hacer un checklist en esta sección.

Si no se utiliza, esta sección se puede eliminar.

# Checklist:

- [ ] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [ ] Se corrieron todas las pruebas y pasaron.
- [ ] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [ ] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

**Comment by iqbalcodes6602** [comment]
@heduenas hello can u please review the changes i have changes the text as you said in the comments.

File: frontend/www/js/omegaup/components/problem/SearchBar.vue [hunk 1]
```diff
@@ -138,7 +138,4 @@ export default class ProblemSearchBar extends Vue {
   }
 }
 
-[data-problem-keyword-search] {
-  z-index: 9999;
-}
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/problem/SearchBar.vue:143]: I think this change doesn't correspond to this PR, could you remove it?

File: frontend/templates/pt.lang [hunk 2]
```diff
@@ -361,7 +361,7 @@ contestNewFormDifferentStartsDesc = "Quando é ativado, indica o tempo em minuto
 contestNewFormEndDate = "Data de término"
 contestNewFormEndDateDesc = "A hora (no horário local) em que o concurso vai acabar."
 contestNewFormForTeams = "Concurso para equipes"
-contestNewFormForTeamsDesc = "Se ativado, só pode ser acessado por equipes em vez de identidades. As equipes devem ser adicionadas a um grupo de equipes."
+contestNewFormForTeamsDesc = "Se ativado, só pode ser acessado por equipes em vez de identidades. As equipes devem ser adicionadas a um grupo de equipes. Esta opción sólo está disponible para nuevas competiciones."
```
Comment (pabo99) [inline frontend/templates/pt.lang:364]: In portuguese would be something like:

```suggestion
contestNewFormForTeamsDesc = "Se ativado, só pode ser acessado por equipes em vez de identidades. As equipes devem ser adicionadas a um grupo de equipes. Esta opção está disponível apenas para novos concursos.."
```

**Review by pabo99 — COMMENTED** [review]
I've left you some suggestions.

Also, you need to run the linter command, it will help you to propagate the changes in all the translations files:

```bash
./stuff/lint.sh
```

**Comment by pabo99** [comment]
One more thing:

I've noticed that when creating a new PR, it includes commits from previous PRs. If you want to start fresh with each PR, you'll need to run the following commands:

```bash
git checkout -b name-of-the-branch upstream/main
```

and then:

```bash
git push origin name-of-the-branch
```

Make sure to replace name-of-the-branch with the appropriate branch name. 

You can check your remotes using:

```bash
git remote -v
```

Here's an example output:

```
origin	https://github.com/iqbalcodes6602/omegaup (fetch)
origin	https://github.com/iqbalcodes6602/omegaup (push)
upstream	https://github.com/omegaup/omegaup (fetch)
upstream	https://github.com/omegaup/omegaup (push)
```

This will help ensure that each PR starts with a clean slate.

File: frontend/www/js/omegaup/components/problem/SearchBar.vue (hunk 1 above)
Comment (iqbalcodes6602) [reply frontend/www/js/omegaup/components/problem/SearchBar.vue:143]: ya definitely

**Comment by iqbalcodes6602** [comment]
hello @pabo99  sir. 
As you said I have done the required changes. Sorry though because I am not familiar with Spanish or Portuguese, but I did the exact changes as told here-

image

Anyways i have done as you told.

**Comment by pabo99** [comment]
Excellent! Now, please execute the following command to generate all the changes in the translation files:

```bash
./stuff/lint.sh
```

**Review by pabo99 — APPROVED** [review]
LGTM!

**Pull request merged**